base/int.jl

```diff
@@ -332,6 +332,21 @@ for to in BitInteger_types, from in (BitInteger_types..., Bool)
     end
 end
 
+"""
+    rem(x, T)
+    mod(x, T)
+    %(x, T)
+
+For an integer type `T`, finds `y::T` such that `x` ≡ `y` (mod n), where n is the
```

Comment:
I think it should be "find" rather than "finds," since we typically try to stick to the imperative when describing functionality in docstrings.